src/components/Calendar/Gutter.js

```diff
@@ -0,0 +1,36 @@
+import React, { memo, forwardRef } from 'react';
+import { Stack } from './Stack';
+import { range } from '../../utils/range';
+import { addMinutes } from 'date-fns';
+
+function GutterComponent(
+  { slotsToRender, sizeOfOneSlot, startTime, step, render },
+  ref
+) {
+  return (
+    <Stack
+      ref={ref}
+      direction={Stack.direction.VERTICAL}
+      distribution={Stack.distribution.NORMAL}
+      className="SkedifyCalendar__Gutter"
```

Comment:
Sometimes we repeat the parent, like for example the `WeekHeader_Days_Day` class.
But here we don't repeat the `SkedifyCalendar__Main` (`SkedifyCalendar__Main__Gutter`). 

It would become very large very quickly, but is there a reason we don't repeat it here?